zap/src/main/java/org/zaproxy/zap/CommandLineBootstrap.java

```
@@ -86,6 +88,23 @@ public int start() {
             } else if (getArgs().isDisplaySupportInfo()) {
                 System.out.println(ZapSupportUtils.getAll(false));
 
+            } else if (sbomFile != null) {
+                try {
+                    int count = ZapSupportUtils.saveSbmoZip(sbomFile);
+                    if (count == 0) {
+                        System.out.println(
+                                Constant.messages.getString("support.savesbom.warn.nosboms"));
+                    } else {
+                        System.out.println(
+                                Constant.messages.getString(
+                                        "support.savesbom.info.generated",
+                                        count,
+                                        sbomFile.getAbsolutePath()));
+                    }
+                } catch (Exception e) {
```

Could let the exception bubble up.